### DISCUSSION TITLE
How is it possible to choose language at startup?

### DISCUSSION THREAD

**Discussion by davinhun** [YetiForceCompany/YetiForceCRM#16925] [How to] [answered]
I installed 2 language on my YetiCRM and I would like to choose a language before login like in the gitdeveloper environment.
How can I add this languge selector into the login page?

![image](https://user-images.githubusercontent.com/20862325/221322212-6be8b1da-53da-4f9c-9861-a21aa092d779.png)

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
https://github.com/YetiForceCompany/YetiForceCRM/issues/2216

**Comment by davinhun** [comment]
Thanks, working
actually this variable is in the main.php not at /config
